ibis/expr/window.py

```diff
@@ -163,6 +203,12 @@ def _validate_frame(self):
                 "'how' must be 'rows' or 'range', got {}".format(self.how)
             )
 
+        if self.max_lookback is not None:
+            if not isinstance(self.max_lookback, ir.IntervalValue):
```

Comment:
I'll make a follow up for this, but I think we might also want to support passing in `pandas.Timedelta` objects here.